### PR TITLE
Initial work

### PR DESCRIPTION
**Pull request by sd-lueckel**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
+*.retry
```
Comment (mattagohni) [inline .gitignore]: here a empty last line is missing

File: tests/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+- hosts: all
+
+  roles:
+    - ansible-role-phpbrew
```
Comment (mattagohni) [inline tests/playbook.yml]: same here

**Review by mattagohni — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (sd-lueckel) [reply .gitignore]: done

File: tests/playbook.yml (hunk 2 above)
Comment (sd-lueckel) [reply tests/playbook.yml]: done

**Review by mattagohni — APPROVED** [review]

**Review by twojtylak — APPROVED** [review]

**Review by michz — APPROVED** [review]

**Pull request merged**